.changeset/little-insects-pull.md

```diff
@@ -0,0 +1,7 @@
+---
+"@marigold/docs": minor
+"@marigold/components": minor
```

Comment:
Adding a new component is just a minor change?

Reply:
yes. major means breaking change. minor = new functionality